web/modules/weather_i18n/translations/en.po

```diff
@@ -75,7 +75,7 @@ msgid "daily-forecast.labels.today.01"
 msgstr "Today"
 
 msgid "daily-forecast.text.chance-precip.01"
-msgstr "chance of precipitation"
+msgstr "Chance of precipitation"
```

Comment:
Sorry for causing the merge conflicts here! This string is used in several places, with different capitalization needs, so I split it into different keys.

Reply:
No problem, thanks for explaining!